types/wordpress__components/button/index.d.ts

```diff
@@ -3,9 +3,9 @@ import { ComponentType, HTMLProps } from '@wordpress/element';
 declare namespace Button {
     interface BaseProps {
         /**
-         * Renders a default button style.
+         * Renders a secondary button style.
          */
-        isDefault?: boolean;
```

Comment:
Rather than replacing it, should we add and deprecated, like the package has done?

Reply:
This is an excellent idea, I have pushed the corresponding commit,